.github/workflows/gatekeeper.yml

```diff
@@ -6,7 +6,7 @@ permissions:
 
 jobs:
   gk-test:
-    name: "Gatekeeper Test"
+    name: "[Informative] Gatekeeper Test"
```

Comment:
should we add `continue-on-error: true`

Reply:
I would think we still want it to fail and see what is causing the error. If it is changes that are limited to frameworks and test fails, then we may not want to merge those changes. For example, refactoring in frameworks should not break GK test.

Reply:
you are right. I didn't know that it would show the gatekeeper_test job as succeed.